service-deployment/README.md

```diff
@@ -120,15 +120,21 @@ If your service need more information, please add your property to cluster confi
 The new property should be placed in clusterinfo, machineinfo or machinelist.
 
 #### Test your service
```

Comment:
>Test [](start = 5, length = 4)

do you mean "(Re)Deploying on individual service"?

Reply:
Yes, both test and redeploy individual service